BASE/src/Microsoft.ApplicationInsights/Channel/Transmission.cs

```diff
@@ -139,6 +141,32 @@ public ICollection<ITelemetry> TelemetryItems
             get; private set;
         }
 
+        /// <summary>
+        /// Gets the flush async id for the transmission.
+        /// </summary>
+        /// [EditorBrowsable(EditorBrowsableState.Never)]
+        public long FlushAsyncId { get; } = Interlocked.Increment(ref flushAsyncCounter);
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this transmission is created from FlushAsync API call.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool HasFlushTask { get; set; } = false;
```

Comment:
nit:
The name doesn't match the summary.. `IsCreatedFromFlush` matches the summary.

(Or is the summary actually incorrect?)

Reply:
Apt name would be `IsFlushAsyncInProgress`. Changed both summary and property name.